Cartesian_kernel/include/CGAL/Cartesian/Aff_transformation_rep_3.h

```diff
@@ -127,6 +131,12 @@ class Aff_transformation_repC3
                        t31 * v.x() + t32 * v.y() + t33 * v.z());
   }
 
+  virtual Plane_3 transform(const Plane_3& p) const
```

Comment:
`virtual Plane_3 transform(const Plane_3& p) const = 0;` ?

Reply:
Oh sorry this is the subclass

Reply:
In fact I think the code which is currently in Aff_transformation_3   for transforming the plane should go here.